### DISCUSSION TITLE
Help request - Error setting up entry Homematic for homematicip_local

### DISCUSSION THREAD

**Discussion by Jensbe1337** [SukramJ/aiohomematic#1898] [Q&A] [answered]
Hi,

I have a problem with my Home Assistant + custom_homematic and my CCU 3 running as a Homematic server.
I think this might be a configuration issue, thus I did not open a bug report, but I am not able to fix this on my own. I studied the docs, FAQ'S and googled a lot, still no progress - so your help is very much appreciated.

I am running the following Home Assistant OS versions in a proxmox environment:

        Core: 2024.12.1
        Supervisor: 2024.11.4
        Operating System: 14.0
        Frontend: 20241127.6

        HACS: 2.0.1
        custom_homematic: 1.73.0

        CCU3 3.79.6

I installed and deinstalled custom_homematic three times now, connect/deconnected the CCU several times, but I am still getting this Error in the logs. I honestly do not know what I can do anymore to fix it. Maybe someone has a good hint where to look at. I will include the full Home Assistant log.

Thank you very much in advance!

[home-assistant_2024-12-07T17-26-38.384Z.log](https://github.com/user-attachments/files/18049303/home-assistant_2024-12-07T17-26-38.384Z.log)

**Comment by SukramJ** [comment, marked as answer]
Looks like one of your CCU sysvars has a faulthy description.

Your options: Use a version prior to 1.73.0 or cleanup (remove non ascii characters) your CCU sysvar descriptions.

**Reply by Jensbe1337** [reply]
Thank you! It was a variable description that contained " ". 
Running like a charm with that being fixed.

**Comment by 21-Jailbraik-21** [comment]
![Screenshot 11 12 2024 um 05 19 11 AM](https://github.com/user-attachments/assets/8ae07db0-d07c-4d24-b8e0-7fca69fc6aab)

That was probably the problem with the bracket.
But I can't remove it because it immediately reappears.
It is set by the system. And not changeable.

**Reply by SukramJ** [reply]
These Brackets are not the Problem.

As already mentioned in the Issue:

`Bezeichnungen dürfen keines der folgenden Zeichen enthalten: <, >, ', ", &, $, [, ], {, } und \`

That is also valid for descriptions.

**Reply by 21-Jailbraik-21** [reply]
In all descriptions “Program/Actuators/Devices”?

**Reply by SukramJ** [reply]
Ain't we are talking about sysvars and programs?

**Reply by 21-Jailbraik-21** [reply]
![Screenshot 11 12 2024 um 06 16 19 AM](https://github.com/user-attachments/assets/a0210336-e322-4d06-a96f-e4238fc8927a)


OK - Danke

**Comment by SukramJ** [comment]
Siehe auch #1906
